templates_apprunner/templates_cspm_eventbridge/FullInstall.yaml

```diff
@@ -0,0 +1,87 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: IAM Role for Agentless
+Parameters:
+  SysdigRoleName:
+    Type: String
+    Default: "SysdigAgentlessRole"
+    Description: Unique role for monitoring AWS accounts
+  SysdigExternalID:
+    Type: String
+    Description: Sysdig ExternalID required for the policy creation
+  SysdigTrustedIdentity:
+    Type: String
+    Description: Sysdig Trusted identity required for policy creation
+  SysdigEBName:
+    Type: String
+    Default: "sysdig"
+    Description: Unique name for role and policy
+  SysdigExternalID:
```

Comment:
As caught by linting, this is a duplicate param

Reply:
removed the duplicate